recipes/ginkgo/all/conandata.yml

```diff
@@ -0,0 +1,10 @@
+sources:
+  "1.3.0":
+    sha256: 1b0e907b4046cdf7cef16d1730c12ba812b38f2764f49f74f454239a27f63596
+    url: https://github.com/ginkgo-project/ginkgo/archive/v1.3.0.tar.gz
+patches:
+  "1.3.0":
+    - patch_file: "patches/cmake-fixes.patch"
+      base_path: "source_subfolder"
+    - patch_file: "patches/windows-iterator.patch"
```

Comment:
I'd love to know where this patch comes from? Did you consider reporting the issue to upstream?

Reply:
These are changes made by myself (I am a part of the Ginkgo development team), and they are already part of upstream or waiting to be merged.

Reply:
Thanks, for the information... Please add a link to the PR so if there's a bump we know it's not required

#3903 and it's PR if you need more details =)

Reply:
windows-iterator.patch is https://github.com/ginkgo-project/ginkgo/pull/665, cmake-fixes.patch is a simplified subset of https://github.com/ginkgo-project/ginkgo/pull/720. Should I link them here in the comments/PR description or as a comment in the YAML file?

Reply:
Please add them at the top of the patch file 🙏 

If you plan on doing a release shortly, perhaps waiting for the next one might be an option?

Reply:
We don't have a timeframe for the next release yet, so I would prefer if we could keep the patches for now.